client/app/l10n/en.json

```diff
@@ -115,7 +115,7 @@
   "Completed submissions": "Completed submissions",
   "Configuration": "Configuration",
   "Configuration completed": "Configuration completed",
-  "Configuration completed. You can review the settings or enable it now.": "Configuration completed. You can review the settings or enable it now.",
+  "Configuration completed. You can review the settings or enable it now.": "Configured. Review or enable the settings.",
```

Comment:
I understand the simplification but in this case we would like to be more explicit as the configuration is composed by multiple steps that all have been completed